src/sanitizer.js

```diff
@@ -106,6 +107,10 @@ const WHITELISTED_ATTRS = [
   'href',
   'on',
   'placeholder',
+  /* Attributes added for amp-bind */
+  // TODO(kmh287): Add more whitelisted attributes for bind?
+  'text',
+  'class',
```

Comment:
Interestingly enough, running the new test on master shows that the sanitizer allows the class attribute and allows values that match the blacklisted regex.